src/nodedef.h

```diff
@@ -422,4 +427,3 @@ class NodeResolver {
 
 
 #endif
-
```

Comment:
The C (and C++) standard requires that the last line of a file is a newline.  I doubt many compilers would fail to work (but they should emit a warning), but it just good to do(TM).